axum-macros/src/from_request/attr.rs

```diff
@@ -73,11 +79,18 @@ pub(crate) fn parse_container_attrs(
                     out_rejection_derive = Some((idx, rejection_derive, opt_outs));
                 }
             }
+            ContainerAttr::Rejection { rejection, path } => {
+                if out_rejection.is_some() {
+                    return Err(double_attr_error("rejection", rejection));
+                } else {
+                    out_rejection = Some((idx, rejection, path));
+                }
+            }
         }
     }
 
-    match (out_via, out_rejection_derive) {
-        (Some((via_idx, via, _)), Some((rejection_derive_idx, rejection_derive, _))) => {
+    match (out_via, out_rejection_derive, out_rejection) {
```

Comment:
This is getting out of hand 🙃 should probably be structured differently to be easier to extend.

Reply:
I'd say it's still manageable. But with the next attribute it will be bad ^^